runtime/src/bank.rs

```diff
@@ -1697,7 +1697,7 @@ impl Bank {
                 new.inherit_specially_retained_account_fields(account),
             )
         });
-
+        new.fill_missing_sysvar_cache_entries();
```

Comment:
this is a new invocation to cover this new code-path (ref: https://github.com/solana-labs/solana/pull/30158/files#r1098222823 #30158)